packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,154 @@
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'scope=openid profile offline_access'
+```
+
+The POST request passes the following parameters:
+
+* `client_id`, which matches the Client ID of the OAuth 2.0 application that you created
+* `scope` to specify which access privileges are being requested for access tokens. See [Scopes](https://developer.okta.com/docs/reference/api/oidc/#scopes) for a list of supported scopes.
+
+**Response example**
+
+```json
+{
+   "device_code": "f5ec0430-c2fe-4a93-912d-08631f1effed",
+   "user_code": "HMBSRJRC",
+   "verification_uri": "https://${yourOktaDomain}/activate",
+   "expires_in": 600,
+   "interval": 5
+}
+```
+
+The properties in the response are:
+
+* `device_code`: The long string that the device uses to exchange for an access token.
+* `user_code`: The text that you enter at the URL that is listed as the value for `verification_uri`.
+* `verification_uri`: The URL that the user needs to enter into their phone to start signing in.
+* `expires_in`: The number of seconds that this set of values is valid. After the device code and user code expire, the user has to start the device verification process over.
+* `interval`: The number of seconds that the device should wait between polling to see if the user has finished signing in.
+
+The `user_code` and `verification_uri` must appear on the smart device for the user.
+
+> **Note:** A QR code compliant `verification_uri` is coming soon.
+
+#### Example of the display on the smart device
+
+![Verification on the smart device:](/img/DeviceAuthGrant2.png)
+
+### Request for user access, ID, and refresh tokens
+
+To retrieve tokens for the user, the smart device needs to make a request to the `/token` endpoint.
+
+**Request example**
+
+```bash
+  curl --location --request POST 'https://${yourOktaDomain}/oauth2/default/v1/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'grant_type=urn:ietf:params:oauth:grant-type:device_code' \
+  --data-urlencode 'device_code={deviceCode)' \
+```
+
+Note the paramters that are being passed:
+
+* `grant_type`: Identifies the mechanism that Okta uses to retrieve the tokens. Value: `urn:ietf:params:oauth:grant-type:device_code`
+* `device_code`: The string that the device uses to exchange for an access token. Use the `device_code` value from the device verification response.
+
+**Response example**
+
+Okta returns a pending response if the user doesn't complete the authentication.
+
+```json
+  {
+     "error": "authorization_pending",
+     "error_description": "The device authorization is pending. Please try again later."
+  }
+```
+
+After the user completes the authentication process by visiting the `/activate` URL and follow the instructions on their device to get an activation code, the token is returned in the response from the `/token` endpoint.
```

Comment:
```suggestion
After the user visits the `/activate` URL and follow the instructions on their device to input the activation code and completes the authentication and authorization, the tokens are returned in the response from the `/token` endpoint.
```
Additionally, we've not called the `user_code` an `activation code` before now. Can we stick to one terminology?